updating/index.adoc

```diff
@@ -0,0 +1,67 @@
+
+[id="updating-clusters-overview-update-cluster-using-web-console"]
+== Updating a cluster using the web console
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster within a minor version using the web console]: You can update an {product-title} cluster within a minor version by using the web console. For more information, see the following items:
+
+* xref:../updating/updating-cluster-within-minor.adoc#update-upgrading-web_updating-cluster-within-minor[Updating a cluster by using the web console]
+* xref:../updating/updating-cluster-within-minor.adoc#update-changing-update-server-web_updating-cluster-within-minor[Changing the update server by using the web console]
+
+[id="updating-clusters-overview-update-cluster-using-cli"]
+== Updating a cluster within a minor version using the CLI
+xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version using the CLI]: You can update an {product-title} cluster within a minor version by using the `oc`.  For more information, see the following procedures:
+
+* xref:../updating/updating-cluster-cli.adoc#update-upgrading-cli_updating-cluster-cli[Updating a cluster by using the CLI]
+* xref:../updating/updating-cluster-cli.adoc#update-changing-update-server-cli_updating-cluster-cli[Changing the update server by using the CLI]
+
+[id="updating-clusters-overview-update-cluster-with-rhel-compute-machines"]
+== Updating a cluster that includes {op-system-base} compute machines
+xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes {op-system-base} compute machines]: If your cluster contains {op-system-base-full} machines, you must perform additional steps to update those machines. You can perform the following sections:
+
+* xref:../updating/updating-cluster-rhel-compute.adoc#update-upgrading-web_updating-cluster-rhel-compute[Updating a cluster by using the web console]
+* xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute-hooks[Optional: Adding hooks to perform Ansible tasks on {op-system-base} machines]
+* xref:../updating/updating-cluster-rhel-compute.adoc#rhel-compute-updating-minor_updating-cluster-rhel-compute[Updating {op-system-base} compute machines in your cluster]
+
+[id="updating-clusters-overview-update-restricted-network-cluster"]
+== Updating a restricted network cluster
+xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: A restricted network environment is one in which your cluster nodes cannot access the internet. You can update a restricted network cluster by using the OpenShift CLI (oc). You can also update a restricted network {product-title} cluster by using the OpenShift Update Service, but you must have an OSUS service running in a cluster. For more information, see the following items:
```

Comment:
This is the first time the "OSUS" acronym is used in this assembly. Is the acronym recommended officially? You could change this to "OpenShift Update Service" for consistency with the other references in the assembly. Alternatively, if it is a recommended product acronym, you could define it in the first instance in each module and the assembly and then use "OSUS" after. I think it would be easier to change this to "OpenShift Update Service".

Reply:
Addressed